addon/components/bs-form.js

```diff
@@ -195,6 +210,14 @@ export default class Form extends Component {
   @defaultValue
   groupComponent = 'bs-form/group';
 
+  /**
+   * @property submitButtonComponent
+   * @type {String}
+   * @private
+   */
+  @defaultValue
+  submitButtonComponent = 'bs-button';
```

Comment:
This reminds me that we should get rid of that pattern sooner than later to make component tree-shakable. It's ok for now, as we have that all over the place, but I just created this issue to not forget this: https://github.com/kaliber5/ember-bootstrap/issues/1206